app/components/search_bar/search_bar.ios.js

```diff
@@ -62,84 +51,59 @@ export default class SearchBarIos extends Component {
     };
 
     onCancel = () => {
-        return new Promise((resolve) => {
-            Keyboard.dismiss();
-            InteractionManager.runAfterInteractions(() => {
-                if (this.props.onCancelButtonPress) {
-                    this.props.onCancelButtonPress();
-                }
-                resolve();
-            });
+        Keyboard.dismiss();
+        InteractionManager.runAfterInteractions(() => {
+            if (this.props.onCancelButtonPress) {
+                this.props.onCancelButtonPress();
```

Comment:
Minor nit, but could these be added to default props so that the function is always there instead of having to check for it?